get-started.md

```diff
@@ -0,0 +1,24 @@
+# How to get started in the Grafana Labs bug bounty program
```

Comment:
This feels a little lengthy to me, I think you could go with something more concise, like:

> Getting started with our bug bounty program